### PR TITLE
[MIG][17.0] l10n_ch_qr_no_amount: Migration to 17.0

### PR DESCRIPTION
**Pull request by sonhd91**
[QR-bill-IV_2024.pdf](https://github.com/OCA/l10n-switzerland/files/15186016/QR-bill-IV_2024.pdf)

### CODE REVIEW

**Review by yankinmax — APPROVED** [review]

**Review by TDu — APPROVED** [review]

File: l10n_ch_qr_no_amount/report/swissqr_no_amount_report.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+
+class ReportSwissNoAmountQR(models.AbstractModel):
+    _name = "report.l10n_ch_qr_no_amount.qr_no_amount_report_main"
+    _inherit = "report.l10n_ch.qr_report_main"
+    _description = "QR-No-Amount-bill"
+
+    @api.model
+    def _get_report_values(self, docids, data=None):
+        return super()._get_report_values(docids, data)
```
Comment (TDu) [inline l10n_ch_qr_no_amount/report/swissqr_no_amount_report.py]: Actually is this needed ?

Comment (ivantodorovich) [reply l10n_ch_qr_no_amount/report/swissqr_no_amount_report.py]: Seems not

Comment (sonhd91) [reply l10n_ch_qr_no_amount/report/swissqr_no_amount_report.py]: Indeed, removed

**Review by TDu — APPROVED** [review]

File: l10n_ch_qr_no_amount/report/swissqr_no_amount_report.py (hunk 1 above)
Comment (TDu) [reply l10n_ch_qr_no_amount/report/swissqr_no_amount_report.py]: thank you

**Pull request merged**